test/manual/api/structure_signature_api_tests.sh

```
@@ -0,0 +1,462 @@
+    exit 1
+  fi
+}
+
+# ─── STRUCTURE TESTS ─────────────────────────────────────────────
+create_structure() {
+  local sys_info=$(get_first_system)
+  local system_uuid=$(echo "$sys_info" | cut -d: -f1)
+  local eve_system_id=$(echo "$sys_info" | cut -d: -f2)
+  local system_name=$(echo "$sys_info" | cut -d: -f3-)
+  echo "==== Creating Structure in system $system_name ($eve_system_id, $system_uuid) ===="
+  local payload=$(jq -n --arg sid "$eve_system_id" --arg name "$system_name" '{
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)